### DISCUSSION TITLE
Model-based evaluations triggered by observations

### DISCUSSION THREAD

**Discussion by marcklingen** [langfuse/langfuse#3918] [Ideas]
### Describe the feature or potential improvement

Run model-based evaluations in Langfuse on single observations within a trace. This is helpful when e.g. wanting to evaluate every function call within a larger agent.

**Comment by kappamax** [comment]
Thanks Marc. This will be a very helpful feature, right now we might have over a dozen generations within a single trace, and we'd like to target a specific one.
